src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String content, String filePath) {
+        try {
+            Files.write(Paths.get(filePath), content.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Error writing file: " + filePath, e);
```

Comment:
It's good that you're rethrowing a RuntimeException with an informative message, adhering to the checklist guidelines.